source/agora/consensus/protocol/Nominator.d

```diff
@@ -873,13 +873,6 @@ extern(D):
             return ValidationLevel.kInvalidValue;
         }
 
```

Comment:
there were 2 problems with this code:
1. we check for the same thing in the ValidateConsensusData too
2. making the check here would have caused us not to record the missing validators, so I couldn't have implemented my ticket

Because of 2., it is not in a separate commit